src/main/java/org/cryptimeleon/craco/sig/sps/agho11/SPSAGHO11SignatureScheme.java

```diff
@@ -25,6 +25,9 @@
  * CRYPTO 2011: Advances in Cryptology – CRYPTO 2011 pp. 649-666
  * https://www.iacr.org/archive/crypto2011/68410646/68410646.pdf
  *
+ * Note: For short messages (k_M = 0 or k_N \in {0,1}) messages and keys will be padded to length
+ * to ensure the scheme's security as described in [1]
```

Comment:
could you specify the cases in more detail, please.

Reply:
and how the padding is then done and if it is transparent to the user. Thanks a lot.

Reply:
More details have been added. Hopefully, this illustrates the situation better?

Reply:
d0f46bbbeb79f09f1053bf2a30d161e90ec3b700 for later reference.